src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const sortList = (listElement) => {
+  const list = [...listElement.children];
+
+  const getSalary = (salaryStr) => salaryStr.slice(1).split(',').join('');
```

Comment:
Probably it is not a sortList scoped function. So move it to the same level as `sortList` and `getEmployees` functions